jsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "checkJs": true,
+    "checkJs": false,
```

Comment:
that probably slipped in by mistake?

Reply:
I turned it off because there is a lot of warning that are not relevant but yes: it shouldn't be in this PR.

Reply:
fixed in 1c483ea